addons/l10n_ar_website_sale/controllers/main.py

```diff
@@ -0,0 +1,56 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import _
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+from odoo.http import request, route
+
+
+class L10nARWebsiteSale(WebsiteSale):
+
+    def _get_mandatory_billing_fields(self):
+        """Extend mandatory fields to add new identification and responsibility fields when company is argentina"""
+        res = super()._get_mandatory_billing_fields()
+        if request.website.sudo().company_id.country_id.code == "AR":
+            res += ["l10n_latam_identification_type_id", "l10n_ar_afip_responsibility_type_id", "vat"]
+        return res
+
+    @route()
```

Comment:
Do we need this to inherit the same route?  (just wondering)

Reply:
@jco-odoo  Yes we needed. Eith the decorator as it is is automatically inheriting for the same route route as the original method.. if we remove the decorator then we will have an error like this one (see bellow).  Please let me know if this does not answer your question

![image](https://user-images.githubusercontent.com/7593953/90696650-073fb500-e253-11ea-8e40-f1caa01fdde1.png)